community/modules/scripts/wait-for-startup/variables.tf

```diff
@@ -17,6 +17,13 @@
 variable "instance_name" {
   description = "Name of the instance we are waiting for"
   type        = string
+  default     = null
+}
+
+variable "instance_names" {
+  description = "A list of names of the instances we are waiting for"
```

Comment:
One of these descriptions should mention that we merge with the other.

Reply:
Assuming we get this to work. I would just say something along the lines of, "the module will wait for all VMs provided to either `instance_name` or `instance_names`." 

Basically make it clear we are checking both.